.woodpecker.yml

```diff
@@ -3,16 +3,36 @@ steps:
     image: tmknom/prettier:3.2.5
     commands:
       - prettier -c .
+    when:
+      - event: pull_request
+
+  build_typescript_plugin:
+    image: node:23-alpine
+    commands:
+      - wget --quiet https://github.com/extism/js-pdk/releases/download/v1.4.0/extism-js-x86_64-linux-v1.4.0.gz -O extism-js.gz
+      - gzip -d extism-js.gz
+      - chmod +x extism-js
+      - export "PATH=$PATH:$(pwd)"
+      - echo $PATH
+      - ls -lah
+      - ls -lah "$(pwd)/extism-js"
+      - apk add --no-cache typescript
+      - cd plugins/typescript_push_webhook
+      - npm install
+      - npm run build
+    when:
+      - event: [pull_request, tag]
```

Comment:
Having some trouble with CI here, somehow its not finding the binary even though its correctly added to PATH. 

https://woodpecker.join-lemmy.org/repos/147/pipeline/52/5